pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_assigning-metadata-to-a-module.adoc

```diff
@@ -1,30 +1,31 @@
 [id="assigning-metadata-to-a-module_{context}"]
-= Assigning metadata to a module
+= Assigning metadata to {ContentTerm}
 
-Assign metadata to a module to enable search engine optimization (SEO). SEO improves the findability of content.
+[role="_abstract"]
+Assign metadata to {ContentTerm} to enable search engine optimization (SEO). SEO improves the findability of content.
```

Comment:
Accurate, but assigning metadata also influences the URL that your doc is presented from. Maybe that still falls under the bucket of SEO? Up to you if you'd like to tweak the sentence.

Reply:
@benradey does assigning the vanity URL affect the URL or do the other metadata also affect the URL?
(Sorry for late response!)

Reply:
Currently, the vanity URL is ignored (although we ask for it anyway because we'll support it "Soon™"), but the Product and Product Version that you assign from the Edit Metadata screen do influence the URL.

For example, just by looking at this URL:
https://access.redhat.com/documentation/en-us/red-hat-enterprise-linux/8/guide/44bfea51-a0b3-4b87-836a-47705ecb7af6
...I can see that the Product is "red-hat-enterprise-linux" (probably named something more human-friendly in Pantheon) and the Product Version is "8" and both those things are assigned through Edit Metadata.

The other metadata fields don't affect the URL at all. So all in all, it's a pretty minor point but wanted to bring it up anyway for completeness' sake. Totally up to you whether it's worth following up on.

Reply:
Thanks @benradey, added the sentence "Assigned metadata also determines the URL at which the {ContentTerm} is published."

Reply:
Cool, sounds great to me!